tamr_client/operation.py

```diff
@@ -19,6 +19,25 @@ class NotFound(TamrClientException):
     pass
 
 
+class Failed(TamrClientException):
+    """Raised when checking a failed operation.
+    """
+
+    pass
+
+
+def check(operation: Operation):
+    """Check if operation was successful and raises an exception if it failed.
+
+    Raises:
+        Failed: If the operation failed.
+    """
+    if not succeeded(operation):
```

Comment:
Need to figure out how to handle `PENDING` and other statuses. Maybe `check` should `wait` until the operation is resolved, and then make sure it `succeeded`?

Reply:
Agreed. Implemented (1) and ready for re-review!